docs/reference/ccr/getting-started.asciidoc

```diff
@@ -57,6 +57,17 @@ response time
 In this guide, you'll learn how to:
 
 * Configure a <<modules-remote-clusters,remote cluster>> with a leader index
+
+[IMPORTANT]
```

Comment:
I feel like this note belongs in the third bullet of the [prerequisites](https://elasticsearch_70186.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/ccr-getting-started.html#ccr-getting-started-prerequisites) where we're telling users which privileges they need on the local cluster. This is an important step, so I think it should be part of the setup for the task itself instead of in the introduction.